docs/guide/sections/dom-binding.md

```diff
@@ -1,6 +1,71 @@
-Full documentation for the DOM Binding module is coming soon.
+Cart.js ships with support for [Rivets.js][], which allows for some powerful templating approaches with minimal effort. Rivets.js allows you to describe in your markup how your data should be rendered, and then handles automatically updating the DOM when that model data changes. With Cart.js, this allows us to describe a template for our cart that will be automatically updated whenever the cart changes.
 
-In the meantime, you can check out the example on the [front page][] in combination with the [Rivets.js documentation][] to get an idea of how things work.
+You can see an example of this in action [at the end of this section].
+
+### Getting Started
+The Rivets module of Cart.js will bind to an element with the attribute `data-cart-view="data-cart-view"`, which needs to wrap all cart elements that need to be dynamically updated. The basic syntax for Rivets templates involves adding an `rv-` attribute to the element, and passing in a reference to the `cart` object that Cart.js has bound with Rivets. 
+
+<div class="callout callout-success">
+  <p style="margin-bottom: 0;">
+    Cart.js binds the `cart` data model to Rivets by default, however you can pass in other data models as the [rivetsModels] Cart.js initialisation option.
+  </p>
+</div>
+ 
+### Rivets Binders
+Binders are core of Rivets' power, and provide for both rendering content into the template, and applying conditional logic to the content.
+
+For example, if we wanted to keep a `<p>` element updated with the number of items in the cart, we could add the `rv-text="cart.item_count"` attribute like so:
+
+```html
+<div data-cart-view="data-cart-view">  
+  <p rv-text="cart.item_count"></p>
+</div>  
+```
+
+This would set the contents of the `<p>` element to the current value of `cart.item_count`, and update this content whenever `cart` changes.
+See Rivets' [Binder Reference][] for a full breakdown of available `rv-` tags. 
+
+
+Rivets also provides an alternative to `rv-text` that enables interpolating content into an element, using a syntax of single curly-braces (not to be confused with Liquid's {% raw %}`{{`{% endraw %}) - the previous example could also be stated as:
+ 
+```html
+<div data-cart-view="data-cart-view">  
+ <p>{ cart.item_count }</p>
+</div>  
+``` 
+
+A full breakdown of Rivets' available binders can be found under the Rivets docs [Binder Reference][].
```

Comment:
A couple more binder examples would be good here, especially around `rv-each="cart.items"`.